src/ClassApplication/Controllers/ValuesController.cs

```diff
@@ -20,18 +21,43 @@ public ClassController(IHttpClientFactory factory, DatabaseService databaseServi
         }
 
         /// <summary>
-        /// This is how you document code
         /// 
-        /// Visit the microservice's endpoint and append /swagger
-        /// to see your docs in action
+        /// Creates a classroom in the database from a given Classroom object
+        /// 
+        /// </summary>
+        /// <param name="classroom">Has a ClassId, Name, and Description</param>
+        /// <returns>An array containing a value determined by the parameter</returns>
+        [HttpPost]
+        public async Task<ActionResult<string>> CreateClassroom(Classroom classroom)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// 
+        /// Returns data from a classroom from a given classroom ID
+        ///
         /// </summary>
-        /// <param name="id">Some sort of Id</param>
+        /// <param name="id">classroom id</param>
         /// <returns>An array containing a value determined by the parameter</returns>
         [HttpGet("{id}")]
-        public async Task<ActionResult<string>> GetValues(string id)
+        public async Task<ActionResult<string>> GetClassroom(string id)
         {
-            await this.classesContainer.CreateItemAsync<dynamic>(new Object() { });
-            return ("Created an object");
+            throw new NotImplementedException();
         }
+
+        /// <summary>
+        /// 
+        /// Finds a classroom by the classrooms name
+        ///
+        /// </summary>
+        /// <param name="className">classroom name</param>
+        /// <returns>An array containing a value determined by the parameter</returns>
+        [HttpGet("{className}")]

```

Comment:
This is actually a conflicting path for the method defined on lines 43 and 44.

How does your controller know how to route

`http://api.notesapp/api/class/algorithms`

to this method and not to the `GetClassroom` method? The paths looks similar.

I would also suggest learning how to use Query Parameters as that is the proper way to search for a document using something other that an ID. Great work though!